cfpb/consumerfinance.gov
# Change search and submit a complaint links to accessible colors

## Changes

- Make the global search hover state darker to be accessible.
- Make the submit a complaint link darker to be accessible and the same as other links.


## How to test this PR

1. Pull branch and `gulp clean && gulp build`
2. Check the global search and submit a complaint links in the header of any page and compare to production.


## Screenshots

Before:

<img width="305" alt="Screen Shot 2021-10-15 at 5 36 11 PM" src="https://user-images.githubusercontent.com/704760/137556294-3970cebb-ad18-4368-882d-b7fdd4dd8e93.png">
<img width="305" alt="Screen Shot 2021-10-15 at 5 36 18 PM" src="https://user-images.githubusercontent.com/704760/137556296-6178b37d-2e9b-4ede-aa9a-7ff51576d90a.png">

After:

<img width="325" alt="Screen Shot 2021-10-15 at 5 34 30 PM" src="https://user-images.githubusercontent.com/704760/137556311-5e36cf1c-8676-406d-9252-e9dbe78fdc01.png">
<img width="332" alt="Screen Shot 2021-10-15 at 5 34 35 PM" src="https://user-images.githubusercontent.com/704760/137556312-9c4a1277-287f-4915-a471-4c55992d3961.png">
